src/Files.App/Views/ModernShellPage.xaml.cs

```diff
@@ -356,21 +366,21 @@ public override void NavigateToPath(string? navigationPath, Type? sourcePageType
 
 				NavigationTransitionInfo transition = new SuppressNavigationTransitionInfo();
 
-				if (sourcePageType == typeof(WidgetsPage)
-					|| ItemDisplayFrame.Content.GetType() == typeof(WidgetsPage) &&
+				if (sourcePageType == typeof(HomePage) ||
```

Comment:
Why was the page renamed?

Reply:
**Before**
- ViewModel: YourHomeViewModel
- Page: WidgetsPage

**After**
- ViewModel: HomeViewModel
- Page: HomePage

**Validation**
- This page is used as Home page
- Should not limit the page scope to Widgets

Reply:
> This page is used as Home page

It used to be called HomePage but it was repurposed as Widgets, if anything the ViewModel should be renamed to WidgetsViewModel. The reasoning behind this is that we might allow users to change the home page in the future.

Reply:
Understood.